sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/configuration/AadOAuth2ClientConfiguration.java

```diff
@@ -101,7 +104,7 @@ OAuth2ClientAuthenticationJwkResolver oAuth2ClientAuthenticationJwkResolver(AadA
     @Bean
     @ConditionalOnMissingBean
     OAuth2AuthorizedClientManager authorizedClientManager(
-            RestOperations aadAuthRestOperations,
+            @Qualifier(AZURE_AD_ACCESS_TOKEN_RETRIEVER_REST_OPERATIONS_BEAN_NAME) RestOperations restOperations,
```

Comment:
Inject this bean through the constructor. We don't need to add qualifier everywhere in this class.